acquisitions/src/main/resources/thunderjet/consortia/features/pieces-api-test-for-cross-tenant-envs.feature

```diff
@@ -0,0 +1,523 @@
+Feature: Pieces API tests for cross-tenant envs
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+
+    * def targetTenant = universityTenant;
+    * def centralHeaders = { 'Content-Type': 'application/json', 'Authtoken-Refresh-Cache': 'true', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(centralTenant)', 'Accept': 'application/json' }
+    * def targetHeaders = { 'Content-Type': 'application/json', 'Authtoken-Refresh-Cache': 'true', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(targetTenant)', 'Accept': 'application/json' }
```

Comment:
I suggest to use one header and choose appropriate tenant in each request to be more clear which tenant request is being send to 
```
 Background:
    * url baseUrl
    * call login consortiaAdmin
    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json', 'Authtoken-Refresh-Cache': 'true' }

  @Negative
  Scenario: Attempt to POST a sharingInstance with invalid payload
    # cases for 400
    # attempt to create a sharingInstance when both source and target tenants are member tenants
    Given path 'consortia', consortiumId, 'sharing/instances'
    **And header x-okapi-tenant = centralTenant**
    And request
    """
    {
      instanceIdentifier: '111841e3-e6fb-4191-8fd8-5674a5107c33',
      sourceTenantId:  '#(universityTenant)',
      targetTenantId:  '#(universityTenant)'
    }
    """
    When method POST
    Then status 400
    And match response == { errors: [{message: "Both 'sourceTenantId' and 'targetTenantId' cannot be member tenants.", type: '-1', code: 'VALIDATION_ERROR'}] }

    Given path 'consortia', consortiumId, 'sharing/instances'
    **And header x-okapi-tenant = universityTenant**
    And request
    """
    {
      instanceIdentifier: '111841e3-e6fb-4191-8fd8-5674a5107c33',
      sourceTenantId:  '#(universityTenant)',
      targetTenantId:  '#(universityTenant)'
    }
    """
    When method POST
    Then status 400
    And match response == { errors: [{message: "Both 'sourceTenantId' and 'targetTenantId' cannot be member tenants.", type: '-1', code: 'VALIDATION_ERROR'}] }
```

Reply:
if I want to know which tenant this requets is being sent to, I have to look at couple of scenarios above.

Reply:
Replaced all occasions